payloads/ready.go

```diff
@@ -45,6 +45,10 @@ type Ready struct {
 	// cpu[0-9]+ entries in /proc/stat.
 	CpusOnline int `yaml:"cpus_online"`
 
+	// Array containing one entry for each network interface present on the
+	// CN/NN
+	Networks []NetworkStat
```

Comment:
I'm not sure management makes sense to track.  It would make sense to have the launcher not report any/all networks, but rather just the compute networks.  I'm not sure if that's already what it does or if it reports any/all networks.  @markdryan what all is in the data member I grabbed out of controller?